src/test/java/rest/salesforce/org/steps/AssetsApiSteps.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ *
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package rest.salesforce.org.steps;
+
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+
+public class AssetsApiSteps {
```

Comment:
You have so many classes that is going to duplicate the same code. I am going to merge now. but you need to delete them.